val.py

```diff
@@ -422,8 +422,8 @@ def run(
             if single_cls:
                 pred[:, 5] = 0
             predn = pred.clone()
-            pred_clone = pred.clone()
-            scale_boxes(im[si].shape[1:], predn[:, :4], shape, shapes[si][1])  # native-space pred
+            pred_clone = pred.clone() # TODO maybe just use predn
+            scale_boxes(im[si].shape[1:], predn[:, :4], shape, shapes[si][1])  # native-space pred # TODO this is doing nothing
```

Comment:
do we need these comments?

Reply:
A thanks for pointing this out. I want to discuss this with you